src/rcc/enable.rs

```diff
@@ -0,0 +1,185 @@
+use super::*;
+use crate::bb;
+
+macro_rules! bus {
+    ($($PER:ident => ($apbX:ty, $bit:literal),)+) => {
+        $(
+			impl Sealed for crate::pac::$PER {}
```

Comment:
Weird spacing (is this a rustfmt oddity?)

Reply:
Fixed.